src/commands/flinkStatements.ts

```diff
@@ -15,91 +16,25 @@ import { showErrorNotificationWithButtons } from "../notifications";
 import { flinkComputePoolQuickPick } from "../quickpicks/flinkComputePools";
 import { flinkDatabaseQuickpick } from "../quickpicks/kafkaClusters";
 import { uriQuickpick } from "../quickpicks/uris";
+import { getSidecar } from "../sidecar";
 import { UriMetadataKeys } from "../storage/constants";
 import { ResourceManager } from "../storage/resourceManager";
 import { UserEvent, logUsage } from "../telemetry/events";
 import { getEditorOrFileContents } from "../utils/file";
 import { FlinkStatementsViewProvider } from "../viewProviders/flinkStatements";
+import { handleWebviewMessage } from "../webview/comms/comms";
 import {
   FlinkSpecProperties,
+  FlinkStatementWebviewPanelCache,
   IFlinkStatementSubmitParameters,
   determineFlinkStatementName,
   localTimezoneOffset,
   submitFlinkStatement,
+  waitForStatementRunning,
 } from "./utils/flinkStatements";
 
 const logger = new Logger("commands.flinkStatements");
 
-/** Poll period in millis to check whether statement has reached results-viewable state */
```

Comment:
These and the related utilities used by these commands moved into corresponding commands utils module.